src/Controller/ForgotPasswordController.php

```diff
@@ -29,8 +30,15 @@ class ForgotPasswordController extends AbstractController
     private const SESSION_TOKEN_KEY = 'forgot_password_token';
     private const SESSION_CAN_CHECK_EMAIL = 'forgot_password_check_email';
 
-    /** @TODO this value should be generated/retrieved from the config... */
-    private const LIFETIME_HOURS = 1;
+    /**
+     * @var ResetPasswordHelperInterface
+     */
```

Comment:
I wouldn't generate this into the user's code... it's subjective... but phpstorm already knows what instance this is thanks to the constructor.